ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    src:
+      paths:
+        - destination_dir: .
+          source_path: /go/src/github.com/redhat-developer/jenkins-operator/.
+  to: jenkins-operator
+#promotion:
+#  name: "4.2"
+#  namespace: ocp
```

Comment:
with no tests and no promotion i'm not sure what this job is going to do?

Reply:
just added unit tests